### PR TITLE
Fixes #49 - FVdycoreCubed_GridComp  moved to separate repository.

### LINKED ISSUE #49: Split FVdycoreCubed_GridComp into sepratate Git repo
This is to support running FV standalone in a virtual machine, but will also facilitate various CTM applications that do not need the full GEOS GCM.

### PR DESCRIPTION
**Pull request by tclune**
This is to support standalone demonstration/benchmarking as well as simplifying usage in CTM's.

Have verified zero diff for short C12 resolution run.     I believe this level of testing is sufficient for what is essentially just a slight change in the directory name from `FVdycoreCubed_GridComp` to `@FVdycoreCubed_GridComp`.

### CODE REVIEW

**Comment by sdrabenh** [comment]
@tclune and @lltakacs, after removing the conflicted file above and re-running checkout_externals in GEOSgcm_GridComp, the executable was successfully built from the current PR. All zero-diff tests passed so these changes seem okay as expected. However, the conflicted file does not appear to have been updated in the new FVdycoreCubed_GridComp repository. @tclune would you mind updating fv_tracer2d.F90 according to hotfix/kgerheis/#13-fv_tracer2d-ksplt? At that point I think it is safe to merge this PR into develop.

**Comment by tclune** [comment]
Ah - I checked it only on the GEOS/develop and the GEOS/MAPL-2.0 branches.  Forgot this was a hotfix and thus needed to be checked on GEOS/master as well.     Have manually fixed it there.

Please confirm, but then approve this pull request.

**Comment by tclune** [comment]
@sdrabenh  Hmm.   I thought "we" fixed the merge conflicts yesterday.     Did you not push them back into this branch?

**Comment by tclune** [comment]
Realized I needed to do a few more things

1.  Issue a new release of GFDL_atmos_cubed_sphere (v1.0.1)
2. Modify Externals.cgf in FVdycoreCubed_GridComp master branch to use v1.0.1
3. Propagate item 2 to develop branch as well
4. Release new FVdycoreCubed_GridComp (v1.0.5)
5. Modify Externals.cfg in this pull request

Simple and clear as mud.

**Comment by mathomp4** [comment]
I saw some oddities in the other repos. We might need to have @tclune make another push for this.

File: Externals.cfg [hunk 1]
```diff
@@ -1,3 +1,11 @@
+[FVdycoreCubed_GridComp]
+required = True
+repo_url = git@github.com:GEOS-ESM/FVdycoreCubed_GridComp.git
+local_path = ./GEOSagcm_GridComp/GEOSsuperdyn_GridComp/@FVdycoreCubed_GridComp
+tag = v1.0.4
```
Comment (mathomp4) [inline Externals.cfg]: Should this be v1.0.5?

Comment (tclune) [reply Externals.cfg]: Yes.  Will fix.

**Comment by tclune** [comment]
The automated tests are failing because the script was not designed to handle the extra checkout_externals step required to pull in the new repos associated with this branch.   I have manually built with gfortran and all is well.

**Comment by tclune** [comment]
Still waiting on the mandatory review by @wmputman

**Pull request merged**